spa/src/pages/InstallationRequested/index.tsx

```diff
@@ -42,6 +42,9 @@ const InstallationRequested = () => {
 			<LoggedinInfo
 				username={username}
 				logout={navigateBackToSteps}
+				onPopupBlocked={() => {
+					//TODO: show proper UI
```

Comment:
is this suppose to be todo?
should we not show popup block alert of this page ?

Reply:
Hi I've changed the comments and leave this out. When popup is blocked, the ui will still be redirected to home page, so actually we don't need to do anything.